lib/smart_answer_flows/uk-benefits-abroad.rb

```diff
@@ -608,11 +609,19 @@ def define
         next_node do |response|
           case response
           when "before_jan_2021"
-            outcome :jsa_eea_going_abroad_maybe_outcome
+            if calculator.benefit == "jsa"
+              outcome :jsa_eea_going_abroad_maybe_outcome
+            elsif calculator.benefit == "winter_fuel_payment"
+              outcome :wfp_going_abroad_eea_maybe_outcome
+            end
           when "after_jan_2021"
```

Comment:
This can be shortened to `when "after_jan_2021", "no"`, to save you having to repeat the same outcome for each case.